packages/calling/src/Metrics/index.test.ts

```diff
@@ -341,6 +343,58 @@ describe('CALLING: Metric tests', () => {
         }
       );
     });
+
+    it('submit bnr enabled success metric', () => {
```

Comment:
Can we keep this as a separate describe section as this is a separate function?

Reply:
addressed